tests/foreman/destructive/test_capsulecontent.py

```diff
@@ -0,0 +1,111 @@
+"""Capsule-Content related tests, which require destructive Satellite
+
+:Requirement: Capsule-Content
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Capsule-Content
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_alpha
+
+from robottelo import constants
+
+pytestmark = [pytest.mark.destructive]
+
+
+@pytest.mark.tier4
+@pytest.mark.skip_if_not_set('capsule')
+def test_positive_sync_without_deadlock(
+    target_sat, large_capsule_configured, function_entitlement_manifest_org
+):
+    """Synchronize one bigger repo published in multiple CVs to a blank Capsule.
+    Assert that the sync task succeeds and no deadlock happens.
+
+    :id: 91c6eec9-a582-46ea-9898-bdcaebcea2f0
+
+    :setup:
+        1. A blank external capsule that has not been synced yet (!) with immediate download
+           policy and running multiple (4 and more) pulpcore workers.
+
+    :steps:
+        1. Sync one bigger repository to the Satellite.
+        2. Create a Content View, add the repository and publish it.
+        3. Create several copies of the CV and publish them.
+        4. Add the Library LCE to the Capsule.
+        5. Sync the Capsule.
+
+    :expectedresults:
+        1. Sync passes without deadlock.
+
+    :customerscenario: true
+
+    :BZ: 2062526
+
+    """
+    workers_count = int(
+        large_capsule_configured.execute('ps -u pulp | grep pulpcore-worker | wc -l').stdout
+    )
+    if workers_count < 4:
+        pytest.skip('Not enough pulpcore workers running to test the scenario.')
```

Comment:
Are pulpcore workers depends on the system configuration? if yes, then having a static count of workers does not really make sense ! 

We have to have it dynamic!

Reply:
The pulpcore workers count is derived from the CPUs count, from this point of view we could say it's dynamic. However, the number here defines the minimal threshold we need to hit the deadlock issue and it is rather static - we need at least some minimal count of workers to hit the issue and we don't really care if we have 2 or 20 more extra workers.

As I wrote in some comment before, in our deployments we always have more and I don't expect the count of CPUs needed for capsule to decline, so we could probably even remove this check.

Reply:
@vsedmik Go for removal of it or increase the count ?

Reply:
All right, I choose removal if you insist on it.